src/components/BottomSheet/BottomSheetLabel.js

```diff
@@ -16,30 +16,26 @@ const BottomSheetLabel = ({ variant, children, ...restProps }) => {
     return null;
   }
 
-  switch (variant) {
-    case "alert":
-      return (
-        <StatusBadge
-          {...restProps}
-          label={children}
-          color={colors.alert.base}
-        />
-      );
-    case "positive":
-      return (
-        <StatusBadge
-          {...restProps}
-          label={children}
-          color={colors.positive.base}
-        />
-      );
-    default:
-      return (
-        <LabelText {...restProps} primary>
-          {children}
-        </LabelText>
-      );
+  const colorsMap = {
+    alert: colors.alert.base,
+    positive: colors.positive.base,
+    accent: colors.heliotrope.dark,
+    caution: colors.caution.dark,
+    neutral: colors.onyx.light
+  };
```

Comment:
I would appreciate if you could please replace these `colors` constants with `colorThemes` constants, so it will be simpler to add theming support for this component down the road: https://github.com/ticketmaster/aurora/blob/master/src/theme/colorThemes.js

`colors.alert.base` => `colorThemes.error.base`
`colors.positive.base` => `colorThemes.success.base`
`colors.heliotrope.base` => `colorThemes.accent01.dark`
`colors.caution.dark` => `colorThemes.caution.dark`
`colors.onyx.light` => `colorThemes.onyx.light`